index.html

```diff
@@ -0,0 +1,153 @@
+					</article>
+				</div>
+			</section>
+			
+<!-- Section Portfolio-->
+			<section class="colorBackgroundLight">
+				<a name="Portfolio">
+				<h1 class="fontCursive colorFontSections">Portfolio</h1>
+				</a>
+				<div class="containerFlex">
+					<article class="articleBox">
+						<h2>Project A</h2>
+						<p>"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. </p>
+					</article>
+					<article class="articleBox">
+						<h2>Project B</h2>
+						<p>"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. </p>
+					</article>
+					<article class="articleBox">
+						<h2>Project C</h2>
+						<p>"Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. </p>
+					</article>
+				</div>
+			</section>
+			
+<!-- Section Skills-->
+			<section class="colorBackgroundDark">
+				<a name="Skills">
+				<h1 class="fontCursive colorFontSections">Skills</h1>
+				</a>
+				<div class="containerFlex">
+					<article class="articleBox">
+						<p class="justifyLeft">Currently learning as part of the Mayden web dev bootcamp:</p>
+							<ul class="justifyLeft">
+								<li>HTML & CSS</li>
+								<li>Git & GitHub</li>
+							</ul>
+					</article>
+					<article class="articleBox">
+						<p class="justifyLeft">Previous programming & application design experience </p>
+							<ul class="justifyLeft">
+								<li>Powershell scripting to manipulate Active Directory users & groups and Exchange mailboxes</li>
+								<li>Designed web front end for a resource management system, including 3 different levels of access</li>
+								<li>Learned Java, JSP, JDBC & SQL to create a conference registration website</li>
+							</ul>
+					</article>
+					<article class="articleBox">
+						<p class="justifyLeft">Previous service delivery application experience</p>
+							<ul class="justifyLeft">
+								<li>ServiceNow, Ivanti/HEAT, VMWare, LANDesk, JIRA</li>
+								<li>Windows 10, 7, XP and Server 2008</li>
+								<li>Mac OSX</li>
+								<li>Active Directory, Exchange (hybrid OnPrem & cloud), Azure AD licensing, SharePoint, Skype</li>
+								<li>Kaseya VSA - ITSM software used to manage other computers</li>
+								
+								<li>Supporting corporate endusers in a windows environment, both onsite & field services</li>
+							</ul>
+					</article>
+				</div>
+			</section>
+			
+<!-- Section Contact-->
+			<section class="colorBackgroundLight">
+				<a name="Contact">
+				<h1 class="fontCursive colorFontSections">Contact</h1>
```

Comment:
same as above

Reply:
As above, I've chosen to use h1 for top level headings which will get repeated eg About, Portfolio etc